SomosMasApp/SomosMasApp/App/Shared/Modules/LogIn/Service/LoginService.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LoginService.swift
+//  SomosMasApp
+//
+//  Created by Mariano Battaglia on 26/03/2022.
+//
+
+import Foundation
+import Alamofire
+
+struct LoginService {
+
+    static let shared = LoginService()
```

Comment:
Why do you do this if you are not using it?
Also  why do you need  singletons?
If you don't have an explanation maybe you should delete it